db/migrate/20150818142457_add_vm_template_id_to_vms.rb

```diff
@@ -0,0 +1,5 @@
+class AddVmTemplateIdToVms < ActiveRecord::Migration
+  def change
+    add_column :vms, :vm_template_id, :bigint
```

Comment:
Right ... so it looks like we're already capturing this in the `relationships` table.  We [parse](https://github.com/ManageIQ/manageiq/blob/ee658ec0f607ca3c9e4645aa90903b9f34cbb95a/app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb#L181-L187) the `parent_vm` information (i.e., the image) and [save](https://github.com/ManageIQ/manageiq/blob/ee658ec0f607ca3c9e4645aa90903b9f34cbb95a/app/models/ems_refresh/save_inventory.rb#L126-L141) it as a `"genealogy"` `relationship`.